src/main/webapp/images/svgs/sprite-github.svg

```diff
@@ -1,12 +1,15 @@
 <!-- Sourced from https://github.com/primer/octicons -->
 <svg xmlns="http://www.w3.org/2000/svg">
-    <symbol id="github-logo" viewBox="0 0 24 24">
+    <symbol id="github-logo" viewBox="0 0 16 16">
```

Comment:
This attribute didn't match the path element below, making the icon smaller than it should be.